2013-01-24 11:15:15 Error: [MissingActionException] Action AppController::View() could not be found.
#0 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(AppController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-01-24 11:18:26 Error: [MissingViewException] View file "C:\xampp\htdocs\cakephp\app\View\App\view.ctp" is missing.
#0 C:\xampp\htdocs\cakephp\lib\Cake\View\View.php(467): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(955): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(193): Controller->render()
#3 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(AppController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-24 11:19:42 Error: Fatal Error (1): Class 'AppController' not found in [C:\xampp\htdocs\cakephp\app\Controller\AppController.php, line 34]
2013-01-24 11:19:42 Error: [FatalErrorException] Class 'AppController' not found
#0 C:\xampp\htdocs\cakephp\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Class 'AppContr...', 'C:\xampp\htdocs...', 34)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'AppContr...', 'C:\xampp\htdocs...', 34, Array)
#2 C:\xampp\htdocs\cakephp\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Class 'AppContr...', 'C:\xampp\htdocs...', 34, Array)
#3 C:\xampp\htdocs\cakephp\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-24 11:19:56 Error: [MissingViewException] View file "C:\xampp\htdocs\cakephp\app\View\App\view.ctp" is missing.
#0 C:\xampp\htdocs\cakephp\lib\Cake\View\View.php(467): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(955): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(193): Controller->render()
#3 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(AppController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-24 11:21:11 Error: [MissingControllerException] Controller class PostsController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:21:22 Error: [MissingControllerException] Controller class IndexController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:21:25 Error: [MissingControllerException] Controller class IndexController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:21:51 Error: [MissingControllerException] Controller class PostsController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:24:26 Error: [MissingControllerException] Controller class PostsController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:24:28 Error: [MissingControllerException] Controller class PostsController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:24:41 Error: [MissingControllerException] Controller class PostsController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:24:43 Error: [MissingControllerException] Controller class PostsController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:27:57 Error: [MissingControllerException] Controller class PostsController could not be found.
#0 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-24 11:31:33 Error: [MissingViewException] View file "C:\xampp\htdocs\cakephp\app\View\Posts\view.ctp" is missing.
#0 C:\xampp\htdocs\cakephp\lib\Cake\View\View.php(467): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(955): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(193): Controller->render()
#3 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-24 11:33:36 Error: [MissingViewException] View file "C:\xampp\htdocs\cakephp\app\View\Posts\view.ctp" is missing.
#0 C:\xampp\htdocs\cakephp\lib\Cake\View\View.php(467): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cakephp\lib\Cake\Controller\Controller.php(955): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(193): Controller->render()
#3 C:\xampp\htdocs\cakephp\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\cakephp\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-25 13:13:37 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:13:57 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:15:47 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:15:49 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:15:50 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:17:52 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:17:54 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:18:26 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:18:28 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:18:36 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:19:41 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:19:43 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:23:50 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:24:54 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:25:13 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:25:24 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:25:27 Error: [MissingControllerException] Controller class RateworksController could not be found.
#0 C:\xampp\htdocs\RateWorks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:25:51 Error: [MissingControllerException] Controller class RateworkController could not be found.
#0 C:\xampp\htdocs\RateWork\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:27:17 Error: [MissingControllerException] Controller class RateworkController could not be found.
#0 C:\xampp\htdocs\RateWork\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:27:19 Error: [MissingControllerException] Controller class RateworkController could not be found.
#0 C:\xampp\htdocs\RateWork\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 13:30:48 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\SeitesController.php, line 8]
2013-01-25 13:30:48 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 13:31:48 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\SeitenController.php, line 8]
2013-01-25 13:31:48 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 13:31:51 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\SeitenController.php, line 8]
2013-01-25 13:31:51 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 13:33:47 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\SeitenController.php, line 8]
2013-01-25 13:33:47 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 13:35:00 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\SeitenController.php, line 8]
2013-01-25 13:35:00 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 13:35:01 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\SeitenController.php, line 8]
2013-01-25 13:35:01 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 13:35:58 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\SeitenController.php, line 8]
2013-01-25 13:35:58 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 13:35:59 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\SeitenController.php, line 8]
2013-01-25 13:35:59 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:06:06 Error: [MissingControllerException] Controller class SeitenController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 14:06:14 Error: [MissingControllerException] Controller class ContentController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 14:07:05 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentController.php, line 8]
2013-01-25 14:07:05 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:08:11 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:08:11 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:08:26 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:08:26 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:08:50 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:08:50 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:09:39 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:09:39 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:09:40 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:09:40 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:10:00 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:10:00 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:11:24 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:11:24 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:11:48 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:11:48 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:11:50 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:11:50 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:12:02 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:12:02 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:13:52 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:13:52 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:13:54 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:13:54 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:14:05 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:14:05 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:14:06 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:14:06 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:15:29 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:15:29 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:15:30 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:15:30 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:15:35 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:15:35 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:15:37 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php, line 8]
2013-01-25 14:15:37 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 14:16:01 Error: [MissingTableException] Table contents for model Content was not found in datasource default.
#0 C:\xampp\htdocs\rateworks\lib\Cake\Model\Model.php(3217): Model->setSource('contents')
#1 C:\xampp\htdocs\rateworks\lib\Cake\Model\Model.php(2674): Model->getDataSource()
#2 C:\xampp\htdocs\rateworks\app\Controller\ContentsController.php(8): Model->find('all')
#3 [internal function]: ContentsController->index()
#4 C:\xampp\htdocs\rateworks\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(ContentsController), Array)
#5 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#6 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(ContentsController), Object(CakeRequest), Object(CakeResponse))
#7 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2013-01-25 14:18:59 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\UsersController.php, line 8]
2013-01-25 14:18:59 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 8, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 15:18:53 Error: [MissingControllerException] Controller class RegisterController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 15:20:22 Error: [MissingActionException] Action UsersController::login() could not be found.
#0 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-01-25 15:37:12 Error: Fatal Error (1): Call to a member function create() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\UsersController.php, line 16]
2013-01-25 15:37:12 Error: [FatalErrorException] Call to a member function create() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 16)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 16, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 16, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-25 16:06:01 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'
#0 C:\xampp\htdocs\rateworks\lib\Cake\Model\Datasource\DboSource.php(459): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\rateworks\lib\Cake\Model\Datasource\DboSource.php(425): DboSource->_execute('INSERT INTO `ra...', Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Model\Datasource\DboSource.php(1007): DboSource->execute('INSERT INTO `ra...')
#3 C:\xampp\htdocs\rateworks\lib\Cake\Model\Model.php(1734): DboSource->create(Object(User), Array, Array)
#4 C:\xampp\htdocs\rateworks\app\Controller\UsersController.php(16): Model->save(Array)
#5 [internal function]: UsersController->register()
#6 C:\xampp\htdocs\rateworks\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#7 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#9 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2013-01-25 16:12:14 Error: [MissingActionException] Action UsersController::add() could not be found.
#0 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-01-25 16:29:40 Error: [MissingControllerException] Controller class RegisterController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-25 16:46:25 Error: Fatal Error (4): syntax error, unexpected '}', expecting ',' or ';' in [C:\xampp\htdocs\rateworks\app\Model\User.php, line 10]
2013-01-25 16:46:25 Error: [FatalErrorException] syntax error, unexpected '}', expecting ',' or ';'
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 10)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 10, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 10, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-28 17:31:46 Error: Fatal Error (4): syntax error, unexpected '?>', expecting function (T_FUNCTION) in [C:\xampp\htdocs\rateworks\app\Model\User.php, line 20]
2013-01-28 17:31:46 Error: [FatalErrorException] syntax error, unexpected '?>', expecting function (T_FUNCTION)
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 20)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 20, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 20, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-28 17:37:00 Error: [NotFoundException] Invalid user
#0 [internal function]: UsersController->edit()
#1 C:\xampp\htdocs\rateworks\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-01-28 18:07:22 Error: [MissingTableException] Table entries for model Entry was not found in datasource default.
#0 C:\xampp\htdocs\rateworks\lib\Cake\Model\Model.php(3217): Model->setSource('entries')
#1 C:\xampp\htdocs\rateworks\lib\Cake\Model\Model.php(2674): Model->getDataSource()
#2 C:\xampp\htdocs\rateworks\app\Controller\EntrysController.php(7): Model->find('all')
#3 [internal function]: EntrysController->index()
#4 C:\xampp\htdocs\rateworks\lib\Cake\Controller\Controller.php(485): ReflectionMethod->invokeArgs(Object(EntrysController), Array)
#5 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#6 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(EntrysController), Object(CakeRequest), Object(CakeResponse))
#7 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2013-01-28 18:12:01 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\EntrysController.php, line 7]
2013-01-28 18:12:01 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 7)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 7, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 7, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-28 18:12:05 Error: [MissingControllerException] Controller class EntriesController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:12:21 Error: [MissingControllerException] Controller class EntriesController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:12:22 Error: [MissingControllerException] Controller class EntriesController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:12:42 Error: [MissingControllerException] Controller class EntriesController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:12:43 Error: [MissingControllerException] Controller class EntriesController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:13:03 Error: [MissingControllerException] Controller class EntriesController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:13:33 Error: [MissingControllerException] Controller class EntriesController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:14:05 Error: [MissingControllerException] Controller class EntriesController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:14:28 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\EntriesController.php, line 7]
2013-01-28 18:14:28 Error: [FatalErrorException] Call to a member function find() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 7)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 7, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 7, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-28 18:19:22 Error: Fatal Error (1): Call to a member function create() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\EntriesController.php, line 24]
2013-01-28 18:19:22 Error: [FatalErrorException] Call to a member function create() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 24)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 24, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 24, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-28 18:22:13 Error: Fatal Error (1): Call to a member function findById() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\EntriesController.php, line 15]
2013-01-28 18:22:13 Error: [FatalErrorException] Call to a member function findById() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 15)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 15, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 15, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-28 18:22:50 Error: Fatal Error (1): Call to a member function findById() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\EntriesController.php, line 39]
2013-01-28 18:22:50 Error: [FatalErrorException] Call to a member function findById() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 39)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 39, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 39, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-28 18:45:10 Error: [MissingControllerException] Controller class LoginController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-28 18:50:47 Error: Fatal Error (4): syntax error, unexpected 'public' (T_PUBLIC) in [C:\xampp\htdocs\rateworks\app\Controller\EntriesController.php, line 31]
2013-01-28 18:50:47 Error: [FatalErrorException] syntax error, unexpected 'public' (T_PUBLIC)
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 31, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 31, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-28 18:54:53 Error: Fatal Error (1): Call to a member function isOwnedBy() on a non-object in [C:\xampp\htdocs\rateworks\app\Controller\EntriesController.php, line 76]
2013-01-28 18:54:53 Error: [FatalErrorException] Call to a member function isOwnedBy() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 76)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 76, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 76, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 13:25:58 Error: [MissingControllerException] Controller class UserController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-29 13:26:12 Error: [MissingControllerException] Controller class UserController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-29 13:27:28 Error: [MissingControllerException] Controller class UserController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-29 13:30:27 Error: [MissingControllerException] Controller class UserController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-29 13:38:49 Error: [MissingControllerException] Controller class UserController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-29 13:39:44 Error: [MissingControllerException] Controller class UserController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-29 13:42:56 Error: [MissingControllerException] Controller class UserController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-29 13:43:06 Error: Fatal Error (4): syntax error, unexpected 'login' (T_STRING) in [C:\xampp\htdocs\rateworks\app\View\Entries\index.ctp, line 5]
2013-01-29 13:43:06 Error: [FatalErrorException] syntax error, unexpected 'login' (T_STRING)
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 5)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 5, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 5, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 13:44:08 Error: [MissingActionException] Action EntriesController::$this->redirect("users() could not be found.
#0 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(EntriesController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-01-29 13:45:33 Error: Fatal Error (1): Call to undefined method View::redirect() in [C:\xampp\htdocs\rateworks\app\View\Entries\index.ctp, line 5]
2013-01-29 13:45:33 Error: [FatalErrorException] Call to undefined method View::redirect()
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\xampp\htdocs...', 5)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\xampp\htdocs...', 5, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\xampp\htdocs...', 5, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 13:58:18 Error: [MissingControllerException] Controller class LoginController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-29 14:08:20 Error: Fatal Error (1): Call to a member function check() on a non-object in [C:\xampp\htdocs\rateworks\app\View\Entries\index.ctp, line 6]
2013-01-29 14:08:20 Error: [FatalErrorException] Call to a member function check() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 14:08:41 Error: Fatal Error (1): Call to a member function check() on a non-object in [C:\xampp\htdocs\rateworks\app\View\Entries\index.ctp, line 6]
2013-01-29 14:08:41 Error: [FatalErrorException] Call to a member function check() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 14:08:50 Error: Fatal Error (1): Call to a member function check() on a non-object in [C:\xampp\htdocs\rateworks\app\View\Entries\index.ctp, line 6]
2013-01-29 14:08:50 Error: [FatalErrorException] Call to a member function check() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 14:16:37 Error: Fatal Error (1): Call to a member function check() on a non-object in [C:\xampp\htdocs\rateworks\app\View\Entries\index.ctp, line 6]
2013-01-29 14:16:37 Error: [FatalErrorException] Call to a member function check() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 14:16:58 Error: Fatal Error (1): Call to a member function check() on a non-object in [C:\xampp\htdocs\rateworks\app\View\Entries\index.ctp, line 6]
2013-01-29 14:16:58 Error: [FatalErrorException] Call to a member function check() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 14:17:02 Error: Fatal Error (1): Call to a member function check() on a non-object in [C:\xampp\htdocs\rateworks\app\View\Entries\index.ctp, line 6]
2013-01-29 14:17:02 Error: [FatalErrorException] Call to a member function check() on a non-object
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 6, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 14:31:13 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\rateworks\app\View\Users\login.ctp, line 23]
2013-01-29 14:31:13 Error: [FatalErrorException] syntax error, unexpected ')'
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 23)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 23, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 23, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 14:56:21 Error: [MissingActionException] Action UsersController::r() could not be found.
#0 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-01-29 15:11:07 Error: Fatal Error (1): Call to undefined function login() in [C:\xampp\htdocs\rateworks\app\Controller\UsersController.php, line 56]
2013-01-29 15:11:07 Error: [FatalErrorException] Call to undefined function login()
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\xampp\htdocs...', 56)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\xampp\htdocs...', 56, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\xampp\htdocs...', 56, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 16:04:22 Error: Fatal Error (4): syntax error, unexpected 'public' (T_PUBLIC) in [C:\xampp\htdocs\rateworks\app\Controller\UsersController.php, line 64]
2013-01-29 16:04:22 Error: [FatalErrorException] syntax error, unexpected 'public' (T_PUBLIC)
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 64)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\xampp\htdocs...', 64, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\xampp\htdocs...', 64, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-29 16:06:04 Error: [MissingActionException] Action UsersController::lgin() could not be found.
#0 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\rateworks\lib\Cake\Routing\Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#2 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-01-29 16:16:05 Error: [MissingControllerException] Controller class AnycontrollerController could not be found.
#0 C:\xampp\htdocs\rateworks\app\webroot\index.php(97): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-03-18 14:24:04 Error: Fatal Error (64): Cannot redeclare AppController::$components in [C:\xampp\htdocs\rateworks\app\Controller\AppController.php, line 45]
2013-03-18 14:24:04 Error: [FatalErrorException] Cannot redeclare AppController::$components
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 45)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 45, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 45, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-03-18 14:24:16 Error: Fatal Error (64): Cannot redeclare AppController::$components in [C:\xampp\htdocs\rateworks\app\Controller\AppController.php, line 45]
2013-03-18 14:24:16 Error: [FatalErrorException] Cannot redeclare AppController::$components
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 45)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 45, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 45, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-03-18 14:24:54 Error: Fatal Error (64): Cannot redeclare AppController::$components in [C:\xampp\htdocs\rateworks\app\Controller\AppController.php, line 44]
2013-03-18 14:24:54 Error: [FatalErrorException] Cannot redeclare AppController::$components
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-03-18 14:24:55 Error: Fatal Error (64): Cannot redeclare AppController::$components in [C:\xampp\htdocs\rateworks\app\Controller\AppController.php, line 44]
2013-03-18 14:24:55 Error: [FatalErrorException] Cannot redeclare AppController::$components
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-03-18 14:24:56 Error: Fatal Error (64): Cannot redeclare AppController::$components in [C:\xampp\htdocs\rateworks\app\Controller\AppController.php, line 44]
2013-03-18 14:24:56 Error: [FatalErrorException] Cannot redeclare AppController::$components
#0 C:\xampp\htdocs\rateworks\lib\Cake\Error\ErrorHandler.php(161): ErrorHandler::handleFatalError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44, Array)
#2 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(926): call_user_func('ErrorHandler::h...', 64, 'Cannot redeclar...', 'C:\xampp\htdocs...', 44, Array)
#3 C:\xampp\htdocs\rateworks\lib\Cake\Core\App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
